packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -1,7 +1,6 @@
 // Copyright 2015 The Chromium Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
-
```

Comment:
nit: We usually do have one blank line after the license header, so maybe revert this change?

Reply:
Good catch.  Done.